### PR TITLE
feat: add egctl only client version

### PR DESCRIPTION
**Pull request by Xunzhuo**
xref: #1085

### CODE REVIEW

**Comment by Xunzhuo** [comment]
cc @zirain

File: internal/cmd/egctl/version.go [hunk 1]
```diff
@@ -48,6 +49,8 @@ func NewVersionCommand() *cobra.Command {
 
 	versionCommand.PersistentFlags().StringVarP(&output, "output", "o", yamlOutput, "One of 'yaml' or 'json'")
 
+	versionCommand.PersistentFlags().BoolVarP(&client, "client", "c", false, "If only output client version")
```
Comment (zirain) [inline internal/cmd/egctl/version.go:52]: `client` is not a good name, specifically should not use `c` for short.

what about add `--remote` with default value `true`?

Comment (Xunzhuo) [reply internal/cmd/egctl/version.go:52]: Good idea. Thanks!

File: internal/cmd/egctl/version.go [hunk 2]
```diff
@@ -48,6 +58,8 @@ func NewVersionCommand() *cobra.Command {
 
 	versionCommand.PersistentFlags().StringVarP(&output, "output", "o", yamlOutput, "One of 'yaml' or 'json'")
 
+	versionCommand.PersistentFlags().BoolVarP(&remote, "remote", "r", true, "If retrieve version from remote apiserver.")
```
Comment (zirain) [inline internal/cmd/egctl/version.go:61]:
```suggestion
	versionCommand.PersistentFlags().BoolVarP(&remote, "remote", "r", true, "If true, retrieve version from remote apiserver.")
```

File: internal/cmd/egctl/version.go [hunk 3]
```diff
@@ -32,14 +32,24 @@ const (
...
 	)
 
 	versionCommand := &cobra.Command{
 		Use:     "version",
 		Aliases: []string{"versions", "v"},
 		Short:   "Show version",
+		Example: `  # Show versions of both client and server.
+  egctl version
+
+  # Show versions of both client and server in Json format.
```
Comment (zirain) [inline internal/cmd/egctl/version.go:45]:
```suggestion
  # Show versions of both client and server in JSON format.
```

**Review by zirain — DISMISSED** [review]
lgtm, with small nits.

File: internal/cmd/egctl/version.go (hunk 2 above)
Comment (Xunzhuo) [reply internal/cmd/egctl/version.go]: Done. Thanks!

File: internal/cmd/egctl/version.go (hunk 3 above)
Comment (Xunzhuo) [reply internal/cmd/egctl/version.go]: Done. Thanks!

**Review by zirain — APPROVED** [review]
lgtm, thanks!

**Pull request merged**